### ISSUE TITLE
rescaler in NormAwareEmbedding

### ISSUE THREAD

**Issue by HanGuangXin** [serend1p1ty/SeqNet#24]
Hi, I'am wondering why there is a `rescaler` in class `NormAwareEmbedding` with `affine=True`? Thanks in advance!

**Comment by serend1p1ty** [comment]
`affine=True` is the default value of `nn.BatchNorm1d`

**Comment by HanGuangXin** [comment]
Thanks!
